### PR TITLE
Telemetry: Update notifications code documentation

### PR DESCRIPTION
**Pull request by medikoo**
Follow up to https://github.com/serverless/utils/pull/90

### CODE REVIEW

File: docs/telemetry.md [hunk 1]
```diff
@@ -38,16 +38,17 @@ In addition to telemetry, Serverless Framework occasionally informs you about ne
...
-- `0` - in this mode all notifications will be disabled
-- `1` - in this mode only notifications about minor and major version upgrades will be visible
-- `2` - in this mode all notifications will be visible
+- `off` (in older versions `0`) - No notifications will be shown
+- `upgrades-only` (in older versions `1`) - Only notifications about minor and major version upgrades will be visible
+- `on` (in older versions `2`) - All notifications will be visible
+- `force` (in older versions `3`) - All notifications will be visible and will be presented constantly ignoring the visibility threshold setting
 
-By default, the notifications are turned on (use mode `'2'`), with the exception of CI environments where the default mode is set to `'1'`.
+By default, the notifications are turned on (mode `'on'`), with the exception of CI environments where the default mode is set to `'upgrades-ony'`.
```
Comment (pgrzesik) [inline docs/telemetry.md:48]: There's small typo - it should say `upgrades-only`

**Review by pgrzesik — COMMENTED** [review]
Looks good, I've spotted one typo. Should we issue an update to the `utils` lib version right after merging this and release a new version of the Framework?

**Comment by mnapoli** [comment]
Nice, that will be useful for testing!

File: docs/telemetry.md (hunk 1 above)
Comment (medikoo) [reply docs/telemetry.md]: Good find 👍  Fixed

**Comment by medikoo** [comment]
> Should we issue an update to the utils lib version right after merging this and release a new version of the Framework?

I think it's fine to merge after `@serverless/util` will be released (as that will already be good enough for new npm installations and possibly npm updates).

Also doc still documents old way, and it's an more internal feature, nothing critical

**Review by pgrzesik — APPROVED** [review]
Looks great :+1:

**Pull request merged**